.gitignore

```diff
@@ -7,3 +7,9 @@ mix.lock
 .exenv-version
 doc
 .DS_Store
+
+# asdf
```

Comment:
I don't see why these shouldn't go (I've got an `.env` as well that I don't want committed), but one tip is to add editor or environment variable files to a global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore has more details

Reply:
Ahh ok. Is this just to prevent `.gitignore` pollution?

Reply:
Yup! It also helps prevent you from accidentally committing stuff and generally helps save you from having to edit .gitignores across projects.